### PR TITLE
Firebaseopensource.com addition

### LINKED ISSUE #860: Consider adding to firebaseopensource.com
This is a great project and it surely would be helpful for Firebase Developers who use Test Lab. Consider [adding](https://github.com/firebase/firebaseopensource.com/issues/new?template=new_project.md) it to [firebaseopensource.com](https://firebaseopensource.com) :)

### PR DESCRIPTION
**Pull request by Sloox**
Added .opensource for firebaseopensource
There is still a requirement for the description to be updated with an issue attached to the firebaseopensource github.
Fixes #860 

It will appear on the firebaseopensource.com website
.

## Checklist

- [x] Documented
- [x] release_notes.md updated

### CODE REVIEW

File: .opensource/project.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "flank",
+    "platforms": ["Android", "iOS"],
+    "content": "README.md",
+    "pages" : ["faq.md", "contributing.md", "release_notes.md"],
+    "related": []
+  }
```
Comment (jan-gogo) [inline .opensource/project.json:7]: missing new line

Comment (Sloox) [reply .opensource/project.json:7]: hmmm above or below the bracket? @jan-gogo

Comment (adamfilipow92) [reply .opensource/project.json:7]: below, always add an empty line at the end of the file. You can configure IntelliJ to do it on save, just enable: 
``` Editor->General->Save files->Ensure an empty line at the end of a file on Save ```

Comment (jan-gogo) [reply .opensource/project.json:7]: Below, it is visible if you open `Files changes` [tab](https://github.com/Flank/flank/pull/978/files)

Here you have explanation https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Comment (Sloox) [reply .opensource/project.json]: > Below, it is visible if you open `Files changed` [tab](https://github.com/Flank/flank/pull/978/files)
> 
> Here you have explanation https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Interesting replies, I have had in the past debates about this but have not seen it from that perspective. I added it to intellij on auto save file as was commented (but now deleted) recommendation.

Comment (adamfilipow92) [reply .opensource/project.json]: Sorry for that delete I recommended enable in InteliJ 
```  Editor->General->Save File->Ensure an empty line at the end of a file on Save ```

**Pull request merged**